### PR TITLE
add: docker/local ipfs config

### PR DESCRIPTION
**Pull request by icezohu**

### CODE REVIEW

**Comment by ianhe8x** [comment]
@icezohu Can you update docs [here](https://github.com/subquery/documentation/blob/89135d592dcfcbeaeb065ae356594118fc1fff23/docs/subquery_network/kepler/indexers/install-indexer-locally.md?plain=1#L9) as well?

File: docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ services:
     depends_on:
       "postgres":
         condition: service_healthy
+      "ipfs":
+        condition: service_healthy
```
Comment (ianhe8x) [inline docker-compose.yml:30]: I can't find you added `healthcheck` for ipfs, unless the image has built in it, this might not work.

Comment (icezohu) [reply docker-compose.yml:30]: Yes, the ipfs image has the built in `healthcheck` command.

File: docker-compose.yml [hunk 2]
```diff
@@ -93,6 +96,20 @@ services:
...
+    volumes:
+      - .ipfs/export:/export
+      - .ipfs/data:/data/ipfs
+      - ./ipfs/ipfs.sh:/container-init.d/ipfs.sh
+    ports:
+      - 4001:4001
+      - 4001:4001/udp
+      - 127.0.0.1:5001:5001
+    expose:
+      - 8080
```
Comment (ianhe8x) [inline docker-compose.yml:111]: I'm confused, shouldn't here `5001`?

File: docker-compose.yml [hunk 3]
```diff
@@ -93,6 +96,20 @@ services:
       timeout: 30s
       retries: 5
 
+  ipfs:
+    image: ipfs/kubo:master-2023-07-20-1fe17a4
+    container_name: indexer_ipfs
+    volumes:
+      - .ipfs/export:/export
```
Comment (ianhe8x) [inline docker-compose.yml]: we have a `.data` for files, i think you can use `.data/ipfs` instead of `.ipfs`

**Comment by icezohu** [comment]
> @icezohu Can you update docs [here](https://github.com/subquery/documentation/blob/89135d592dcfcbeaeb065ae356594118fc1fff23/docs/subquery_network/kepler/indexers/install-indexer-locally.md?plain=1#L9) as well?

doc updated as below pr:
https://github.com/subquery/documentation/pull/379/files

File: docker-compose.yml (hunk 3 above)
Comment (icezohu) [reply docker-compose.yml]: `.ipfs` replaced by `.data/ipfs` now

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**